### DISCUSSION TITLE
filenames for .css and .pss files

### DISCUSSION THREAD

**Discussion by Aariq** [EDmodel/ED2#378] [Q&A] [answered]
It seems like there are some rules for how .css and .pss files are named.

If I just make up a file name like "generic.pss" and "generic.css" I get a segfault, but if I take the exact same files and rename them "siteid1000020684.fia2000.radius0.075.lat42.1973lon-124.3494.pss" (the filename of a file I know works), then the simulation begins.

Are the rules for these filenames documented somewhere, and is ED2 using the filenames _as data_?

**Comment by mdietze** [comment, marked as answer]
Correct, ED2 is a gridded model so the lat and lon of each polygon need to be in the filename
https://github.com/EDmodel/ED2/blob/4809be535528cdce5aa84230d99ab70d966bf21e/ED/src/io/ed_read_ed10_20_history.f90#L210
